configurecompiler.cmake

```diff
@@ -477,6 +477,8 @@ if (CLR_CMAKE_PLATFORM_UNIX)
   else()
     add_compile_options(-Wno-unused-variable)
     add_compile_options(-Wno-unused-but-set-variable)
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-discarded-qualifiers")
```

Comment:
Where does this warning fire? Would it better (easy enough) to just fix the problem?

Reply:
src/corefx/System.Globalization.Native/pal_calendarData.c:386:16: warning: passing argument 1 of ‘ures_close_ptr’ discards ‘const’ qualifier from pointer target 
type [-Wdiscarded-qualifiers]
     ures_close(rootResBundle);

Reply:
src/corefx/System.Globalization.Native/pal_calendarData.c:387:16: warning: passing argument 1 of ‘ures_close_ptr’ discards ‘const’ qualifier from pointer target type [-Wdiscarded-qualifiers]
     ures_close(calResBundle);

src/corefx/System.Globalization.Native/pal_calendarData.c:388:16: warning: passing argument 1 of ‘ures_close_ptr’ discards ‘const’ qualifier from pointer target type [-Wdiscarded-qualifiers]
     ures_close(targetCalResBundle);

src/corefx/System.Globalization.Native/pal_calendarData.c:389:16: warning: passing argument 1 of ‘ures_close_ptr’ discards ‘const’ qualifier from pointer target type [-Wdiscarded-qualifiers]
     ures_close(erasColResBundle);

src/corefx/System.Globalization.Native/pal_calendarData.c:390:16: warning: passing argument 1 of ‘ures_close_ptr’ discards ‘const’ qualifier from pointer target type [-Wdiscarded-qualifiers]
     ures_close(erasResBundle);

src/corefx/System.Globalization.Native/pal_calendarData.c:419:22: warning: initialization discards ‘const’ qualifier from pointer target type [-Wdiscarded-qualifiers]
         char* name = GetCalendarName(calendarId);

Reply:
There are a total of 5 warnings.

Reply:
These warnings are highlighting real inconsistencies. They should be easy to fix: 
- The arguments of the `CloseResBundle` method should not be `const`.
- `char* name` local should be `const`

Reply:
done